serverless/serverless
# Update Azure Node.js Template

## What did you implement:

Updates the `azure-nodejs` template to reflect current best practices and enable local running of Azure Functions

## How did you implement it:

Generated an Azure Function via official VS Code extension for Azure Functions and mirrored structure in template directory.

Handler should be placed inside of directory with the name `index.js`. That should be alongside the `function.json` for event bindings. Also added a few extra files for help in running Azure Functions locally for development. See `.vscode/`, `local.settings.json` and others.

Also added the `azure-functions-core-tools` to the `devDependencies` and added a start script to run function locally.

## How can we verify it:

```bash
sls create -t azure-nodejs -p myApp`
cd myApp
npm install
npm start
```

Your Azure Function should be running locally


## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
